components/partials/common/trade/advanced-settings/error.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div
+    v-if="hasWarning || hasError"
+    class="flex mt-4 gap-2 text-xs font-semibold"
+    :class="hasWarning ? 'text-warning' : 'text-error'"
+    font-semibold
+    text-2xs
+  >
+    <IconExclamationCircleFill />
+    <span v-if="hasWarning">
+      {{ slippageWarning }}
+    </span>
+    <span v-if="hasError">
+      {{ slippageError }}
+    </span>
+  </div>
+</template>
+<script lang="ts">
+import Vue from 'vue'
+import { BigNumberInBase } from '@injectivelabs/utils'
+import { TradeError } from 'types/errors'
+
+export default Vue.extend({
+  props: {
+    slippageTolerance: {
+      type: String,
+      required: true
+    },
+
+    tradingTypeMarket: {
+      type: Boolean,
+      required: true
+    }
+  },
+
+  computed: {
+    slippageError(): string | undefined {
```

Comment:
Lets use `slippageTooHighError` directly and remove `slippageError` and `hasError` computed property